adapters/appnexus/appnexus.go

```diff
@@ -595,6 +595,9 @@ func (a *AppNexusAdapter) MakeBids(internalRequest *openrtb.BidRequest, external
 						BidVideo:     impVideo,
 						DealPriority: bidExt.Appnexus.DealPriority,
 					})
+					if len(internalRequest.Cur) > 0 {
+						bidResponse.Currency = internalRequest.Cur[0]
```

Comment:
Shouldn't this be set to `bidResp.Cur`? What's the rationale behind setting it to the currency in the request? Is the Appnexus bidder always guaranteed to bid in the same currency as in the request? Even then I'd say we should be setting it to `bidResp.Cur` rather than `internalRequest.Cur[0]`

Reply:
Great catch. I'm also moving the `if` statement to a place that makes more sense. Test was modified accordingly.